### ISSUE TITLE
Feature request: Command line tool to fetch study metadata/bulkdata to local directory

### ISSUE THREAD

**Issue by wayfarer3130** [dcmjs-org/dicomweb-client#87]
It would be nice to have an example tool that fetched a patient or study to a local directory - ideally in the DICOMweb format in terms of directory names, with options to control what got fetched.  Something like the dcm4che dcmqr command, but using node/local retrieves.

**Comment by pieper** [comment]
Yes, agreed, this would. be very handy.  I started looking at this a while back but never went very far.  A curses based interface could also be interesting to consider.
